src/mpid/ch3/channels/nemesis/src/mpid_nem_lmt_shm.c

```diff
@@ -179,6 +179,24 @@ int MPID_nem_lmt_shm_start_recv(MPIDI_VC_t *vc, MPIR_Request *req, MPL_IOV s_coo
         mpi_errno = MPID_nem_allocate_shm_region(&vc_ch->lmt_copy_buf, vc_ch->lmt_copy_buf_handle);
         if (mpi_errno) MPIR_ERR_POP(mpi_errno);
 
+#ifdef HAVE_HWLOC
+        /* bind shared memory to HBM */
+        if (MPIR_Process.hbmset_is_valid) {
+            int info_flag;
+            char info_value[MPI_MAX_INFO_VAL+1];
+            MPIR_Info_get_impl(MPIR_Process.obj_mbind, "COPYBUFS", MPI_MAX_INFO_VAL, info_value,
```

Comment:
We typically use small letters for info keys.  Also, as a separate cleanup pass, we should probably update all mpich-specific info keys to prefix them with `mpich_`.  Obviously not a part of this PR, but it'll be good to create an issue for that.

Reply:
I will convert it to lower case and create an issue for the rest of the info keys.